source/s2n/s2n_tls_channel_handler.c

```diff
@@ -717,6 +719,38 @@ static int s_parse_protocol_preferences(
     return AWS_OP_SUCCESS;
 }
 
+static size_t s_tl_cleanup_key = 0; /* Address of variable serves as key in hash table */
+static struct aws_event_loop_local_object s_tl_cleanup_object;
```

Comment:
make a note that this same object is inserted into multiple event loops' hash tables